LEGO1/lego/legoomni/include/legoracers.h

```diff
@@ -88,11 +88,21 @@ class LegoRaceCar : public LegoCarRaceActor, public LegoRaceMap {
 
 	static EdgeReference g_edgeReferences[];
 	static const SkeletonKickPhase g_skeletonKickPhases[];
-	static const char* g_strSpeedCopy;
+	static const char* g_strSpeed;
+	static const char* g_srtsl18to29[];
+	static const char* g_srtsl6to10[];
+	static const char* g_emptySoundKeyList[];
+	static const char* g_srtrh[];
 	static const char* g_srt001ra;
 	static const char* g_soundSkel3;
+	static MxU32 g_srtsl18to29Index;
+	static MxU32 g_srtsl6to10Index;
+	static MxU32 g_emptySoundKeyListIndex;
+	static MxU32 g_srtrhIndex;
+	static MxLong g_timeLastSoundPlayed;
 	static MxS32 g_unk0x100f0b88;
 	static MxBool g_unk0x100f0b8c;
+	static Mx3DPointFloat g_vector020;
```

Comment:
I think the unk0x with address naming pattern would be a better fit for now

Reply:
According to Ghidra, this variable is not referenced anywhere else, so
1. it will always be equal to `(0.,2.,0.)`, and
2. we will not get a lot of additional insight into its purpose (other than a better understanding of what `VTable0x94` does semantically).
If you still feel like it should be changed, I can of course do that. Other suggestions for the name are also welcome.

Reply:
> (other than a better understanding of what VTable0x94 does semantically).

Yes, generally speaking we should wait until we have that understanding before we assign a "real" name to a variable. It seems very unlikely that the developers would have named this variable after its contents.

In the meantime, please consider that this is a collaborative project: if someone who is unfamiliar with this code runs into `g_vector020` they are not going to know what to make of that name. They are going to wonder what `vector020` means. Does it clearly and distinctively convey any information which cannot otherwise be trivially obtained by looking at its definition (which already immediately reveals its type and value)?

On the other hand, using the pattern for as-yet-to-be-named variables that we established for this project (`unk` followed by `0x...` address) is clear: it means we don't have a good name for it yet and we need to figure out more about it before we can name it.

I understand that while reverse engineering code, it can sometimes be helpful to give a variable a temporary name such as `vector020` because it makes it easier to work with it in your head while you write the code. But this is more of a "mental crutch" that is very individual to the person who is doing the reverse engineering and should not make it into committed code which will be shared with and read/interpreted by others.